src/components/input/Input.ts

```diff
@@ -1,11 +1,37 @@
 import { EventEmitter } from "@stencil/core"
-import { Color } from "../../model"
+import { isly } from "isly"
+import { Color, Data } from "../../model"
 import { Looks } from "./Looks"
 
-export interface Input {
+export interface InputElement {
+	value?: Data[string]
 	color?: Color
 	name: string
 	looks: Looks
-	smoothlyInput: EventEmitter<Record<number, any>>
-	smoothlyInputLooks: EventEmitter<(looks: Looks, color: Color | undefined) => void>
+}
+export namespace InputElement {
```

Comment:
Make "Element" a namespace under the "Input" namespace, so we get Input.Element for the type, and Input.Element.type for the type, etc